handbook/governance/code-of-ethics.md

```diff
@@ -0,0 +1,43 @@
+
+## Avoided Behaviors
+The following behaviors should be avoided:
+
+* Accepting work from dOrg clients/partners independently without paying a referral fee (20%) to dOrg for sourcing said work
+* Deletion or malicious alteration of communications/comments/messages on dOrg channels except, in cases, where ethical violations have occurred and such action has been approved by dOrg moderators
+* Engagement in abuse, discrimination, disrespect, harassment, or promotion of said or otherwise offensive content against/to any prospective/past/current dOrg client/partner/builder
+* Exertion of personal copyright over any work prepared for, paid for by, or otherwise commissioned by dOrg, or any past/current dOrg client/partner
+* Obstruction of dOrg operations through 'down staking' of DAO proposals except, in cases, where the proposer has admitted that a wish to rescind their proposal and/or a significant enough margin of voters have expressed a wish to change their votes such that the outcome (passing or failing) of the proposal would be otherwise changed
+* Participation in any strike, slowdown, sabotage, or work stoppage in retaliation for the action(s) or inaction of any prospective/past/current dOrg client/partner/builder
+* Public defamation of any prospective/past/current dOrg client/partner/builder without prior authorization and/or through unofficial channels
+
+## Enforcement
+dOrg follows the principle of restorative justice when avoided behaviors occur but builds in certain safeguards to protect the collective as a whole.
+
+1. When an avoided behavior seems to have occured, the offender(s) should first be contacted directly in order to reach an accord. Perhaps, the offender(s) made (a) mistake(s) and are/is prepared to immediately rectify it themselves. 
+2. If this is not possible, those affected should contact dOrg moderators to mitigate any immediate damage that may have occured and attempt to mediate between the victim(s) & the offender(s). A mediation should include sharing experiences of what happened, discusssion of who was harmed by the behavior and how, the creation of a consensus for what the offender(s) can do to repair the harm from the offense(s), and the enaction of said consensus.
+3. If a mediation is successful, then the DAO should pass a 'censure' proposal of the offender(s) which includes a record (private to dOrg members) of the mediation to be reviewed if/when future offense(s) occur. After 'censure' proposals pass or fail, the offender(s) are/is considered to be then in good standing with dOrg and no further action is neccesary.
+4. If a mediation is unsuccessful, then the DAO should weigh the severity of the offense(s) and decide whether deactivation or partial Rep slashing of the offender(s) is warranted.
+
+### Partial Rep Slashing
```

Comment:
I'm not sure I understand in what circumstances partial rep slashing would be appropriate vs. suspension. I personally lean more towards slashing rep as a very last resort that's only taken in the case of expulsion.

Reply:
It seems that without partial rep slashing, the only actions the DAO can take against an individual who may of acted contrary to the Code of Ethics is deactivation. This makes consequences be all or nothing, so having something in between may help making it easier to find the right consequences.

Reply:
Temporary deactivation is one in-between remedy (e.g. removed from discord for 2 weeks)

Reply:
@orishim It's concerning to me to have individuals who are suspended from internal spaces (like Discord) retain Rep within the DAO as I believe there's a risk of retaliation. For example, as long as an individual has Rep, they can engage in a down stake bidding war and potentially freeze our operations. This may not be a long term concern if we move to a different DAO framework but certainly is one in the short term. Is this an acceptable risk for this? If it occurs, would we move to just suspension/expulsion and just attempt to outbid them?

Reply:
I reworked partial rep slashing into its own 'censure' so that there's another intermediary option before suspension available.